applications/nrf_desktop/src/hw_interface/Kconfig.click_detector

```diff
@@ -8,7 +8,7 @@ menu "Click detector configuration"
 
 config DESKTOP_CLICK_DETECTOR_ENABLE
 	bool "Enable click detector"
-	depends on !DESKTOP_BUTTONS_NONE
+	depends on CAF_BUTTONS
```

Comment:
this should actually depends on CAF_BUTTON_EVENTS` as we don't care what is the source of the button events - only that these events are delivered

Reply:
the same for motion and passkey (potentially other places?)

Reply:
Click detector is `nrf_desktop` specific (at least for now) and actually `depends on CAF_BUTTONS`. Loosening this dependency may lead to configuration issues.

Reply:
it depends on the events being sent not the buttons
note that if we add dependency on caf_buttons then user won't be able to use buttons implementation from the application

Reply:
when everything was inside the nrf_desktop it was slightly different as nrf_desktop as a whole knew about all possible implementations

Reply:
Agree. Will change it for all similar places.